### ISSUE TITLE
Account.delete(options) bug

### ISSUE THREAD

**Issue by tanin47** [stripe/stripe-java#435]
```
	public DeletedAccount delete(RequestOptions options)
			throws AuthenticationException, InvalidRequestException,
			APIConnectionException, CardException, APIException {
		return delete(null, null);
	}
```

It doesn't pass `options` to the next method.

https://github.com/stripe/stripe-java/blob/master/src/main/java/com/stripe/model/Account.java#L343

**Comment by ob-stripe** [comment]
Thanks for report @tanin47. I've pushed a fix in #436 and we'll release an updated version of the library soon.

**Comment by tanin47** [comment]
Thank you. It doesn't block me; I merely switched to one of the overloaded methods. Thanks for addressing it promptly.

**Comment by brandur-stripe** [comment]
Thanks for the report! Olivier's patched has been released as part of 5.26.0.
